### ISSUE TITLE
[PM-204] misplaced spaces in error reports look unprofessional

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#324]
2010.09.21 14:37:01.566 PDT: [ERROR]  Error in file:///home/voeckler/Pegasus/futuregrid/condor-staging/config/sites.xml at line 72at column 126 :The reference to entity "amp" must end with the ';' delimiter.

[1] There needs to be a space after the line number and before the continuation text.
[2] There must not be a space before the colon (only after).

**Reporter:** @voeckler
**Resolution:** Won't Fix
**Watchers:**
@vahi

**Comment by mayani** [comment]
**Author:** @vahi

not enough context for this
